test/controllers/declaration/DeclarantDetailsControllerSpec.scala

```diff
@@ -103,28 +107,47 @@ class DeclarantDetailsControllerSpec extends ControllerSpec {
       }
     }
 
-    "return 303 (SEE_OTHER) and redirect to is declarant exporter details page" when {
+    "answer is yes" should {
+
+      onJourney(STANDARD, SUPPLEMENTARY, SIMPLIFIED, OCCASIONAL) { request =>
+        "return 303 (SEE_OTHER) and redirect to Consignment References details page" in {
+
+          withNewCaching(request.cacheModel)
+          val correctForm = JsObject(Map(isEoriKey -> JsString(YesNoAnswers.yes)))
```

Comment:
This could be maybe simplified to `Json.obj(isEoriKey -> YesNoAnswers.yes)` in order to remove the `Map`